src/main/resources/assets/paths/datasetV2.yaml

```diff
@@ -1,26 +1,12 @@
 get:
-  deprecated: true
-  summary: Get Datasets
+  summary: Get Streamed Datasets
   description: | 
-    Returns Dataset list filtered by authenticated user role.
-    Admins can see all datasets, both active and inactive.
-    Chairpersons can see all active datasets and inactive ones 
-    associated to any of their Data Access Committees. 
-    All other users can see active datasets.
-    If the optional `asCustodian` parameter is provided, all datasets that the user has created
-    or is specified as a custodian for will be returned.
```

Comment:
Is this initial context no longer relevant?  I know `asCustodian` has been removed but how about the rest?

Reply:
Yes, all of these conditions are no longer valid.